src/prefect/workers/server.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import FastAPI, APIRouter
+from typing import Union
+import uvicorn
+
+from prefect.workers.base import BaseWorker
+from prefect.workers.process import ProcessWorker
+
+
+def start_healthcheck_server(
+    worker: Union[BaseWorker, ProcessWorker], log_level: str = "error"
+) -> None:
+    """
+    Run a healthcheck FastAPI server for a worker.
+
+    Args:
+        worker (BaseWorker | ProcessWorker): the worker to check health for
+        log_level (str): the log level to use for the server
+    """
+    webserver = FastAPI()
+    router = APIRouter()
+
+    router.add_api_route("/health", worker.check_if_worker_is_polling, methods=["GET"])
+    router.add_api_route("/info", worker.get_status, methods=["GET"])
```

Comment:
@desertaxle we can remove this, but i revived this kind of functionality here in case having a `GET /info` introspection is useful

Reply:
I like the idea of getting more information from a worker (in particular, prefect and Python version), but this isn't quite the right interface. If we had workers report info along with their heartbeat, then we could show that info in the UI, but we wouldn't be able to do that with this endpoint because we wouldn't be able to call it. Let's remove this endpoint so that people don't start to depend on it and continue thinking about ways to surface more worker info.

Reply:
makes sense!